pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -171,7 +171,7 @@ func (r *ReconcileDirectVolumeMigrationProgress) Reconcile(request reconcile.Req
 	}
 
 	if !pvProgress.Status.IsReady() {
-		return reconcile.Result{Requeue: true}, nil
+		return reconcile.Result{Requeue: true, RequeueAfter: time.Second * 5}, nil
```

Comment:
I wonder if this if block is necessary? looks like the intent is to requeue at 5 seconds irrespective of the status

Reply:
Yeah maybe not, let me take a look and see if it's safe to remove